files/galaxy/tpv/tools.yml

```
@@ -474,10 +474,14 @@ tools:
 
   toolshed.g2.bx.psu.edu/repos/bgruening/whisperx/whisperx/.*:
     inherits: basic_docker_tool
+    gpus: 1
     cores: 1
     mem: 5
+    params:
+      docker_run_extra_arguments: '--gpus all --env WHISPERX_MODEL_DIR=$WHISPERX_MODEL_DIR --env WHISPERX_DEVICE=$WHISPERX_DEVICE'
     env:
       WHISPERX_MODEL_DIR: "/data/db/models/whisperx/whisperx_models/"
+      WHISPERX_DEVICE: "cuda"
```

This should also be added for more accuracy.
```suggestion
      WHISPERX_DEVICE: "cuda"
      WHISPERX_COMPUTE_TYPE: "float16"
```